public/demo.js

```diff
@@ -173,8 +173,8 @@ $(document).ready(function() {
             return false;
         }
 
-        if (($("#dropdownMenuInput").text()).toLowerCase().indexOf("language") < 0) {
-            var source = getLanguageCode($.trim($("#dropdownMenuInput").text()));
+        if (getSrcLang().toLowerCase().indexOf("language") < 0) {
```

Comment:
Notes below on getSrcLang(), but I think there's a bug here in that getSrcLang() will return "Detect" or "Choose" and leave out the "language" part.